.github/workflows/benchmark.yml

```diff
@@ -61,6 +61,8 @@ jobs:
       - name: Compile benchmarks
         run: cargo bench ${{ matrix.cargo_flags }} --no-run
       - name: Run benchmarks
+        env:
+          RUSTFLAGS: "--cfg heavy_bench"
```

Comment:
How would you feel about "comprehensive_bench"? Or "all_bench"? Or "full_bench"?

I think "full_bench" is a better descriptor of what this is.

Reply:
Either-or; Don't mind. Updated to `full_bench` 👍